app/js/actions/types.js

```diff
@@ -71,3 +76,17 @@ export const checkTypes = () => {
         .then(() => dispatch(refreshValidation()));
     };
 };
+
+
+export const checkImportableTypes = () => {
+    return (dispatch, getState) => {
+        const {
+            connection: { uri, secretKey },
+            superTypes: { importableTypes }
+        } = getState();
+        const url = `http://${uri}/api/types/importabletypes`;
+        // TODO: don't hit the server if there is nothing new to ask...
+        fetchAPI(secretKey, 'POST', url, Array.from(importableTypes))
```

Comment:
You don't need to worry about the body (`Array.from(importableTypes)`) for the request since it is never used by your endpoint and I'm pretty sure you can just omit the argument entirely.